DiagramNewProductPage.jsp

```diff
@@ -0,0 +1,120 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: pepo
+  Date: 07.11.16
+  Time: 16:24
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<html>
```

Comment:
With the CMS this repetition of header and footer will be replaced by pages like accountLayoutPage.jsp, productLayoutPage.jsp, productLayoutPage1.jsp which become something like this: 
```
<template:page pageTitle="${pageTitle}">
    <cms:pageSlot position="Section1" var="comp" element="div" class="productDetailsPageSection1">
        <cms:component component="${comp}" element="div" class="productDetailsPageSection1-component"/>
    </cms:pageSlot>
    <product:productDetailsPanel />
    <cms:pageSlot position="CrossSelling" var="comp" element="div" class="productDetailsPageSectionCrossSelling">
        <cms:component component="${comp}" element="div" class="productDetailsPageSectionCrossSelling-component"/>
    </cms:pageSlot>
    <cms:pageSlot position="Section3" var="comp" element="div" class="productDetailsPageSection3">
        <cms:component component="${comp}" element="div" class="productDetailsPageSection3-component"/>
    </cms:pageSlot>
    <cms:pageSlot position="UpSelling" var="comp" element="div" class="productDetailsPageSectionUpSelling">
        <cms:component component="${comp}" element="div" class="productDetailsPageSectionUpSelling-component"/>
    </cms:pageSlot>
    <product:productPageTabs />
    <cms:pageSlot position="Section4" var="comp" element="div" class="productDetailsPageSection4">
        <cms:component component="${comp}" element="div" class="productDetailsPageSection4-component"/>
    </cms:pageSlot>
</template:page>
```

template:page corresponds to WEB-INF/tags/responsive/template/page.tag  which in turn calls WEB-INF\tags\responsive\template\master.tag.   page.tag and master.tag come from the accelerator and should already be in the generated storefront extension.